### PR TITLE
SR Linux `startup-config` in CLI format

### PR DESCRIPTION
**Pull request by hellt**
This PR makes it possible to provide the startup-config for SR Linux in CLI format.

The same `startup-config` directive is used to provide a path to a file with the following contents:

```
set / system information location test123
```

In that case, containerlab will recognize that the file content is not a JSON object and will not mount this file as the `config.json` file.
Instead, it will apply the instructions in that file during the post-deploy stage via `sr_cli`.

In summary, this is the proposed way to provide overlay configurations to SR Linux nodes in CLI format that will get applied over the default config.

### CODE REVIEW

**Comment by hellt** [comment]
@jbemmel I propose to reuse the startup-config directive instead of creating another node property as you proposed in #702

As to the reflect-based property retrieval I am hesitating to accept this. I try to avoid reflection whenever possible to keep the compile time type cheking. @karimra may be able to provide his more experienced view on that.

Enabling interfaces defined in the topo file is a good idea, I will likely add it as another PR.

**Comment by hellt** [comment]
Yes, that is the tradeoff I can't make peace with. Go is explicit and wordy (compared to python) in that sense, but that is not without its benefits.

So let's keep it the idiomatic for the time being
